### PR TITLE
Run an application's console

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -37,5 +38,11 @@ def status(app):
     _service(app, 'status')
 
 
+@task
+def console(app):
+    """Attach to an application's console"""
```
Comment (jamiecobbett) [inline app.py]: `Attach to a Rails application's console`?

Comment (alexmuller) [inline app.py]: Yep, thanks - fixed!

**Comment by mattbostock** [comment]
:+1: Should we rename the function also to make it specific to Rails?

**Comment by jamiecobbett** [comment]
> :+1: Should we rename the function also to make it specific to Rails?

I think so.

**Comment by alexmuller** [comment]
Updated to `app.rails_console`

**Pull request merged**